src/Parser/GuardianDruid/CombatLogParser.js

```diff
@@ -0,0 +1,263 @@
+        icon: SPELLS.GORE_BEAR.icon,
+        importance: getIssueImportance(unusedGoreProcs, 0.45, 0.6, true),
+      });
+    }
+    if (unusedGGProcs > 0.30) {
+      results.addIssue({
+        issue: <span>Your <SpellLink id={SPELLS.GALACTIC_GUARDIAN.id} /> procs should be used as soon as you get them so they are not overwritten. You missed {(this.modules.galacticGuardianProcs.GGProcsTotal - this.modules.galacticGuardianProcs.consumedGGProc)}/{(this.modules.galacticGuardianProcs.GGProcsTotal)} procs. ({formatPercentage((this.modules.galacticGuardianProcs.GGProcsTotal - this.modules.galacticGuardianProcs.consumedGGProc) / this.modules.galacticGuardianProcs.GGProcsTotal)} %)</span>,
+        icon: SPELLS.GALACTIC_GUARDIAN.icon,
+        importance: getIssueImportance(unusedGGProcs, 0.45, 0.6, true),
+      });
+    }
+    if (unusedGoEProcs > 0.30) {
+      results.addIssue({
+        issue: <span>Your <SpellLink id={SPELLS.GUARDIAN_OF_ELUNE.id} /> procs should be used as soon as you get them so they are not overwritten. You missed {(this.modules.guardianOfEluneProcs.GoEProcsTotal - this.modules.guardianOfEluneProcs.consumedGoEProc)}/{(this.modules.guardianOfEluneProcs.GoEProcsTotal)} procs. ({formatPercentage((this.modules.guardianOfEluneProcs.GoEProcsTotal - this.modules.guardianOfEluneProcs.consumedGoEProc) / this.modules.guardianOfEluneProcs.GoEProcsTotal)} %)</span>,
+        icon: SPELLS.GUARDIAN_OF_ELUNE.icon,
+        importance: getIssueImportance(unusedGoEProcs, 0.45, 0.6, true),
+      });
+    }
+
+    const castEfficiencyCategories = SPELL_CATEGORY;
+    const castEfficiency = getCastEfficiency(CPM_ABILITIES, this);
+    castEfficiency.forEach((cpm) => {
+      if (cpm.canBeImproved && !cpm.ability.noSuggestion) {
+        results.addIssue({
+          issue: <span>Try to cast <SpellLink id={cpm.ability.spell.id} /> more often ({cpm.casts}/{cpm.maxCasts} casts: {Math.round(cpm.castEfficiency * 100)}% cast efficiency). {cpm.ability.extraSuggestion || ''}</span>,
+          icon: cpm.ability.spell.icon,
+          importance: cpm.ability.importance || getIssueImportance(cpm.castEfficiency, cpm.recommendedCastEfficiency - 0.05, cpm.recommendedCastEfficiency - 0.15),
+        });
+      }
+    });
+
+    results.statistics = [
+      <StatisticBox
+        icon={(
+          <img
+            src="/img/healing.png"
+            style={{ border: 0 }}
+            alt="Healing"
+          />
+        )}
+        value={`${formatNumber(this.totalHealing / this.fightDuration * 1000)} HPS`}
+        label={(
+          <dfn data-tip={`The total healing done was ${formatThousands(this.totalHealing)}.`}>
+            Healing done
```

Comment:
Might a DTPS display be more appropriate as first statistic? I reckon comparing that between tanks of the same class will usually show more of their performance relatively than the other stats.